ui/app/src/components/ActivityDashlet/ActivityDashlet.tsx

```diff
@@ -538,6 +538,7 @@ export function ActivityDashlet(props: ActivityDashletProps) {
             </SizedTimelineSeparator>
             <TimelineContent sx={emptyTimelineContentSx} />
           </CustomTimelineItem>
+          {/* @ts-expect-error InfiniteScroll cannot be used as a JSX component. */}
```

Comment:
@rart error still shows after updating typescript to latest, and there are no updates for 'react-infinite-scroller' or '@types/react-infinite-scroller'